.gitignore

```diff
@@ -7,3 +7,4 @@ target/
 .settings/org.eclipse.core.resources.prefs
 .settings/org.eclipse.jdt.core.prefs
 .vscode/launch.json
+.vscode/settings.json
```

Comment:
should we just ignore the whole .vscode/ folder and the whole .settings folder?

Reply:
i think that is better

Reply:
the .vscode folder is now being ignored